src/dotnet/commands/dotnet-help/HelpCommand.cs

```diff
@@ -49,14 +53,40 @@ public class HelpCommand
 
         public static int Run(string[] args)
         {
+
+            CommandLineApplication app = new CommandLineApplication(throwOnUnexpectedArg: false);
+            app.Name = "dotnet help";
+            app.FullName = LocalizableStrings.AppFullName;
+            app.Description = LocalizableStrings.AppDescription;
+
+            CommandArgument commandNameArgument = app.Argument($"<{LocalizableStrings.CommandArgumentName}>", LocalizableStrings.CommandArgumentDescription);
+
+            app.OnExecute(() => 
+            {
+                Cli.BuiltInCommandMetadata builtIn;
+                if (Cli.BuiltInCommandsCatalog.Commands.TryGetValue(commandNameArgument.Value, out builtIn))
+                {
+                    // var p = Process.Start(GetProcessStartInfo(builtIn));
+                    var process = ConfigureProcess(builtIn.DocLink.ToString());
```

Comment:
Should we make the DocLink a string straight up instead of doing this conversion?